app/views/homescreen/blocks/_administration.html.erb

```diff
@@ -0,0 +1,47 @@
+<h3 class="homescreen-blocks--item-header">
+  <span class="icon-context icon-settings"></span>
+  <%= l(:label_administration) %>
+</h3>
+
+<ul class="homescreen--arrow-links">
```

Comment:
I do not know of any interest in this style. I'd suggest that we make this a styleguide as soon as interest in arrow links spikes due to the amazing grace of the homescreen.